tests/mount.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "mount one container" {
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  run buildah mount "$cid"
+  [ "${status}" -eq 0 ]
+  buildah rm $cid
+  buildah rmi -f alpine
+}
+
+@test "mount bad container" {
+  run buildah mount badcontainer 
+  [ "${status}" -ne 0 ]
+}
+
+@test "list currently mounted containers" {
+  cid1=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  buildah mount "$cid1"
+  cid2=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  buildah mount "$cid2"
+  cid3=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  buildah mount "$cid3"
+  run buildah mount
+  [ "${status}" -eq 0 ]
```

Comment:
It would be good to add a line like:
[[ ${lines[2]} =~ "/var/lib/containers" ]]
To make sure there are at least 3 lines with mounts listed.  I'd really like to verify that 3 lines were returned, but I'm not seeing how to do that offhand.

Reply:
updated, PTAL